data/movie/Stranded (2013)/blu-ray-2013/disc01-summary.txt

```diff
@@ -0,0 +1,37 @@
+Name: Stranded
+Source file name: 00002.mpls
+Duration: 1:27:51
+Chapters count: 8
+Size: 18.8 GB
+Segment count: 1
+Segment map: 0
+Type: MainMovie
+File name: Stranded.mkv
+
+Name: Trailers
+Source file name: 00009.mpls
+Duration: 0:04:40
+Chapters count: 2
+Size: 713.9 MB
+Segment count: 2
+Segment map: 3,4
+Type: Trailer
+File name: Trailers.mkv
+
+Name: Stranded - FX
```

Comment:
This is a very small nitpick but do we need to prefix extras with the name of the movie ("Stranded")? I prefer that these names match what they are actually called on the disc UI. I care less about how the File name field is named.